cms/templates/container.html

```diff
@@ -14,7 +14,7 @@
 )
 from openedx.core.djangolib.markup import HTML, ugettext as _
 %>
-<%block name="title">${xblock.display_name_with_default_escaped} ${xblock_type_display_name(xblock) | h}</%block>
+<%block name="title">${xblock.display_name_with_default | h} ${xblock_type_display_name(xblock) | h}</%block>
```

Comment:
@agaylard - As @mushtaqak pointed out, we are moving toward making all templates "safe by default" by adding a page directive like you can see in index.html in [this PR](https://github.com/edx/edx-platform/pull/11551/files).  Here's [a comment](https://github.com/edx/edx-platform/pull/11551#issuecomment-184759074) about some things to look for if you make these changes.

As I noted earlier, more instructions will be coming and this will be a requirement in the future, but it isn't now.  It's up to you if you want us to help chip away at this.  However, it is important for you (and everyone) to understand that there is nothing special about this particular field, and we will want all fields to be escaped properly. 

Thanks.